interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
```

Comment:
The format of defining the tool and giving examples are for sure ways to express your confidence with the topic. When the examples also showcase how you personally used the tool, it really leaves a great impression upon the interviewer. Use this section to fine tune your conversation with an interviewer.